Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.8
+
+# Copy current sources
+COPY . /src
+
+# install bets
+RUN pip install /src
+
+# install all possibly dependencies for plugins
+RUN pip install pigar
+RUN pigar -p /src/plugins-requirements.txt -P /src
+RUN pip install -r /src/plugins-requirements.txt
+
+# Install some 3rd party plugins
+RUN pip install beets-artistcountry
+RUN pip install beets-copyartifacts3
+RUN pip install git+git://github.com/geigerzaehler/beets-check.git@master
+RUN pip install beets-alternatives
+RUN pip install beets-follow
+RUN pip install beets-noimport
+RUN pip install pip install git+git://github.com/igordertigor/beets-usertag.git@master
+RUN pip install pip install git+git://github.com/8h2a/beets-barcode.git@master
+RUN pip install beets-ydl
```

Comment:
For a Dockerfile we ship with this repository, I would like to _not_ include any third-party plugins. It seems a little strange to "endorse" some but not all… the way we endorse plugins is by including them with the main beets alternatives.

Reply:
I've added all plugins that I can isntall via `pip` and that hadn't beet marked as archived.

Why? Because I'm using `follow` plugging and I did this container mainly for myself :)

Reply:
The simplest option for users to install additional plugins is probably to make their own Dockerfile and extend ours (it'd be good to document this somewhere), as it keeps the official beets image lightweight and impartial to plugins.

Reply:
If I'm not wrong, another option could be use an `ARG` variable, like an env variable to pass this dependencies at image build time, so you can add something like `RUN pip install $BEETS_PLUGINS`